frontend/src/components/assessments/student-experience/question-elements/MultiSelect.tsx

```diff
@@ -16,8 +18,8 @@ const MultiSelect = ({
 
   return (
     <CheckboxGroup
-      onChange={(e) => updateAnswer(e as number[])}
-      value={currentAnswer}
+      onChange={(e) => updateAnswer(stringOrNumberArrayToNumberArray(e))}
+      value={currentAnswer.map(String)}
```

Comment:
Yeah I don't think there's a way around this unfortunately.... Probably the "standard" way to handle this is actually to store things only as strings on the backend (floating point is unreliable 💀) and only convert to `number` if we need to do some sort of numerical manipulation (or perhaps when checking answers, since we'd want to make sure `0.50` is marked as correct for `0.5`).

Reply:
LOL CS370 flashbacks 😂 re: backend update - i'm not sure if making that update is worth the effort right now, so do you think we are okay leaving it as is? or do you think we should create a ticket for the backlog?

Reply:
yeah prob not worth the effort atm, only if it becomes a problem